src/Microsoft.ApplicationInsights/Extensibility/W3C/W3CActivityExtensions.cs

```diff
@@ -29,7 +29,11 @@ public static class W3CActivityExtensions
         /// <param name="activity">Activity to generate W3C context on.</param>
         /// <returns>The same Activity for chaining.</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable CS3001 // Argument type is not CLS-compliant
```

Comment:
nit (and you don't have to fix it): you can list all warnings in one line separated by comma
`#pragma warning disable CS3001, CS3002`